docs/source/api_reference.rst

```diff
@@ -244,30 +244,32 @@ General Transform Primitives
 .. autosummary::
     :toctree: generated/
 
+    AbsoluteDiff
     Absolute
     Cosine
     IsNull
     NaturalLogarithm
     Negate
     Percentile
     RateOfChange
+    SameAsPrevious
     Sine
     SquareRoot
     Tangent
-
+    Variance
 
 Location Transform Primitives
 *****************************
 .. autosummary::
    :toctree: generated/
 
-   CityblockDistance
-   GeoMidpoint
-   Haversine
-   IsInGeoBox
-   Latitude
-   Longitude
-
+    CityblockDistance
+    GeoMidpoint
+    Haversine
+    IsInGeoBox
+    Latitude
+    Longitude
+    Season
```

Comment:
`Season` should be in the Datetime Transform Primitives section

Reply:
Fixed:
https://github.com/alteryx/featuretools/pull/2460/commits/7b162d66842596a2eb47407388ad6c896c7253ce